kiwi/utils/command_capabilities.py

```diff
@@ -103,10 +103,11 @@ def check_version(
         try:
             command = Command.run(arguments)
             for line in command.output.splitlines():
-                match = re.search('[0-9]+(\.[0-9]+)*', line)
-                if match:
+                matches = re.findall(r'([0-9]+(\.[0-9]+)*)', line)
```

Comment:
Are the added `()` necessary?

Reply:
The () are for grouping in the regular expression which makes it easier to get stuff after any "." that are in the version.

Reply:
I have been also surprised by that see the following example:

```python
>>> import re
>>> re.findall(r'([0-9]+(\.[0-9]+)*)', 'grub2-mkconfig (GRUB2) v2.02')
[('2', ''), ('2', ''), ('2.02', '.02')]
>>> re.findall(r'[0-9]+(\.[0-9]+)*', 'grub2-mkconfig (GRUB2) v2.02')
['', '', '.02']
```

Reply:
I just checked: Unlike `re.search`, `re.findall` returns the groups only, not the full match. So the `()` on the outside are indeed necessary.